src/board.py

```diff
@@ -1,8 +1,9 @@
 class Board:
+    SIZE = 9
     border = "⊱ –––––– {⋆⌘⋆} –––––– ⊰",
     divider = "      ---+---+---"
     space = " "
-    
+        
```

Comment:
You have some extra whitespace here. I usually configure my editor to trim any trailing whitespace on save. Helps avoid merge conflicts to just not have trailing whitespace.

Reply:
Yeah I have my normal editors setup to do that, I'm not sure how to do that in vim though, ill have to look into it.